src/components/Notifications/EventLog/EventLogFilter.ts

```diff
@@ -1,9 +1,9 @@
 import { ClearFilters, Filters, SetFilters } from '@redhat-cloud-services/insights-common-typescript';
 
 export enum EventLogFilterColumn {
-    EVENT = 'event',
     APPLICATION = 'application',
-    BUNDLE = 'bundle'
+    EVENT = 'event',
+    BUNDLE = 'bundle',
```

Comment:
I was not sure if we still wanted to keep it for the URL filtering... right now, I have it so that if all the applications are selected for a specific bundle, the bundle shows up in the URL instead of the corresponding applications. I think it is easier to understand IMO. 

However, if not all the applications are listed then I list the bundle and it's applications. This approach does have a drawback, which is since different bundles can have same application names, if you had two bundles selected in the URL and two applications that were present in both bundles, it might filter out the wrong ones depending on which one comes first in the bundles network request.

Reply:
> I was not sure if we still wanted to keep it for the URL filtering... right now, I have it so that if all the applications are selected for a specific bundle, the bundle shows up in the URL instead of the corresponding applications. I think it is easier to understand IMO.

We still want to have the URL filtering; if we do require this for keeping it, then it's fine.

> However, if not all the applications are listed then I list the bundle and it's applications. This approach does have a drawback, which is since different bundles can have same application names, if you had two bundles selected in the URL and two applications that were present in both bundles, it might filter out the wrong ones depending on which one comes first in the bundles network request.

I see, maybe we want to prepend the applications with the bundle and skip the bundle altogether ? I'm not really sure because the url could probably grow a lot. Lets try solving that in a separate PR